ncm-metaconfig/src/main/pan/components/metaconfig/config-common.pan

```diff
@@ -1,17 +0,0 @@
-# ${license-info}
-# ${developer-info}
-# ${author-info}
-# ${build-info}
-
-unique template components/${project.artifactId}/config-common;
-
-include { 'components/${project.artifactId}/schema' };
-
-# Set prefix to root of component configuration.
-prefix '/software/components/${project.artifactId}';
-
-#'version' = '${project.version}';
-#'package' = 'NCM::Component';
```

Comment:
cleanup should not change current behaviour imho. i removed it here because it was commented out.
at some point i would like it as part of https://github.com/quattor/ncm-ncd/issues/41, but now i wouldn't know which "version" to use (`project.version`, `no-snapshot-version`,...).

i looked at the `ncm-cdispd` code, each component is subscribed to its prefix, and if the CCM checksum of the path changes, the component runs. since version is under the prefix, a change in version will provide a different checksum of the prefix, and thus will trigger the component to run.

i always tought that there was a mechanism somewhere that also triggered the component if the component package was updated, but i wouldn't know how that worked, so thinking about it, that probably doesn't happen :smile: (well, maybe :confused: ) i opened https://github.com/quattor/ncm-cdispd/issues/24 to not forget to investigate this